README.md

```diff
@@ -10,18 +10,13 @@ shared.
 
 ## Working on this repo
 
-Use Yarn to install dependencies:
-
-```shell
-yarn install
-```
-
 Common tasks are defined as Make targets:
 
 ```shell
-make lint
-make build
-make test
+make install  # Install dependencies with NPM
+make lint     # Run static analysis
+make build    # Build output files
+make test     # Test
 ```
 
 Node 14 is required to work on this repository, though will not be required for
```

Comment:
You might want to move this to to top and still mention `npm install`

Reply:
Why would `npm install` need mentioned? Generally you shouldn't need to run it at all, since the Makefile handles it automatically.

Reply:
ah, got it :+1: